sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/AsyncSample.java

```diff
@@ -118,5 +120,39 @@ public static void main(String[] args) throws InterruptedException, JsonProcessi
         boolean created = createTwinsSemaphore.tryAcquire(2, 20, TimeUnit.SECONDS);
         System.out.println("Source twins created: " + created);
 
+        String targetTwin1_Id = "targetTwin_1_" + random.nextInt();
+        String targetTwin_1 = "{\"$metadata\": {\"$model\": \"dtmi:samples:HVAC;1\"}, \"Efficiency\": 10, \"TargetTemperature\": 10, \"TargetHumidity\": 10}";
```

Comment:
when we get to the samples code, we will need to read this from the disk.

Reply:
Yes, that's correct.